src/test/rules/no-useless-template-literals_test.ts

```diff
@@ -25,7 +25,8 @@ ruleTester.run('no-useless-template-literals', rule, {
   valid: [
     {code: 'html`foo ${someVar} bar`'},
     {code: 'html`foo bar`'},
-    {code: 'html`<foo .prop=${"literal"}></foo>`'}
+    {code: 'html`<foo .prop=${"literal"}></foo>`'},
+    {code: 'html`<foo\nattr="foo"\n.prop=${"literal"}></foo>`'}
```

Comment:
I'm curious if something like what prettier might do in some cases would cause more issues as well.

Something like: 

```
<wc-foo
  attr="bar"
  .prop=${
    'literal'
  }
></wc-foo>
```